docs/src/Tooltip.doc.js

```diff
@@ -1,165 +1,514 @@
 </Tooltip>
 `}
-  />,
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Use Tooltips to distinguish related actions with visually similar iconography."
+        defaultCode={`
+<Flex gap={1}>
+  <Tooltip text="Align left">
+    <IconButton
+      accessibilityLabel="Align left"
+      bgColor="white"
+      icon="text-align-left"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+  <Tooltip text="Align center">
+    <IconButton
+      accessibilityLabel="Align center"
+      bgColor="white"
+      icon="text-align-center"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+  <Tooltip text="Align right">
+    <IconButton
+      accessibilityLabel="Align right"
+      bgColor="white"
+      icon="text-align-right"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+</Flex>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Don’t use a Tooltip to communicate critical information, such as an error or other interaction feedback."
+        defaultCode={`
+<Tooltip text="Pssst! Looks like you've already saved this Pin.">
+  <Button color="red" text="Save" inline />
+</Tooltip>
+`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Accessibility"
+    description={`Be sure to include an \`accessibilityLabel\` for the screen reader within the child element. While the Tooltip provides a more visible description on screen, its \`text\` property won’t get picked up by the screen reader.
+`}
+  >
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        defaultCode={`
+<Tooltip text="Send">
+  <IconButton
+    accessibilityLabel="Send this Pin"
+    bgColor="white"
+    icon="share"
+    iconColor="darkGray"
+    size="lg"
+  />
+</Tooltip>`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
 );
 
 card(
-  <Example
-    name="Tooltips with link"
-    description="Passing in a link will render the link in the tooltip"
-    defaultCode={`
+  <MainSection name="Variants">
+    <MainSection.Subsection
+      title="Inline"
+      description="Pass this flag in when rendering the tooltip around an inline element such as an Icon,
+      IconButton, or a Button with the inline prop set. This will help correctly position the tooltip
+      alongside the element."
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+<Box borderStyle="sm" padding={2} width={400}>
+  <Tooltip inline text="Send">
+    <IconButton
+      accessibilityLabel="Send this Pin"
+      bgColor="white"
+      icon="share"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+</Box>
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Ideal direction"
+      description="Passing in a link will render the link in the tooltip."
```

Comment:
Copy Pasta? description should be updated

Reply:
DONE